src/mono/sample/HelloWorld/HelloWorld.csproj

```diff
@@ -44,6 +45,10 @@
     </MonoAOTCompiler>
   </Target>
 
+  <ItemGroup>
+      <ProjectReference Include="$(RepoTasksDir)AotCompilerTask\MonoAOTCompiler.csproj" ReferenceOutputAssembly="false" Condition="'$(RunAOTCompilation)' == 'true'" AdditionalProperties="Configuration=Debug"/>
```

Comment:
For my education, why those changes were necessary? I thought we could already run the mono sample with FullAOT

Reply:
`./build.sh mono+libs+tasks` on OSX it doesn't bulid the AOT compiler task anymore because someone added a condition to `tasks.proj` to skip it on non-mobile platforms.  I think the long-term direction for tasks.proj is to get rid of it and add proper project references to all the projects that need tasks